selfdrive/boardd/boardd_setup.py

```diff
@@ -1,10 +1,25 @@
 from distutils.core import Extension, setup
 from Cython.Build import cythonize
+import os
 
 from common.cython_hacks import BuildExtWithoutPlatformSuffix
 
+CROSS_COMPILATION = os.getenv("CROSS_COMPILATION") is not None
+sysroot_args=[]
+
+if CROSS_COMPILATION:
+  os.environ['CC'] = 'aarch64-linux-gnu-gcc'
+  os.environ['CXX'] = 'aarch64-linux-gnu-g++'
+  os.environ['LDSHARED'] = 'aarch64-linux-gnu-gcc -shared'
+  os.environ['LDCXXSHARED'] = 'aarch64-linux-gnu-g++ -shared'
+  os.environ["LD_LIBRARY_PATH"] = "/usr/aarch64-linux-gnu/lib"
+  sysroot_args=['--sysroot', '/usr/aarch64-linux-gnu']
+
```

Comment:
copy/pasting this into every cython extension setup is pretty dirty. can we make these setup files into scons builders?